RELEASE.md

```diff
@@ -0,0 +1,28 @@
+# How to have a release
+
+For Falter-releases, we usally do follow this workflow:
+
```

Comment:
Here, maybe we could add that releases are configured in the falter-common package and specifically the file ```packages/falter-common/files-common/etc/freifunk_release``` is modified and it's contents are used in the image generation process as well as in adding images to selector.berlin.freifunk.net.

Any other information which you think would be useful to express about the falter-common package, here would be a good place to do it.  Also, in the text below we would just need to say "the **freifunk_release**" file

Reply:
done in new version.